### ISSUE TITLE
Better windows narrator support

### ISSUE THREAD

**Issue by Wallby** [ONLYOFFICE/DesktopEditors#1242]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Not a person who uses windows narrator, but I did see another issue about sound and thus decided to test this.

Currently using windows narrator..
.. doesn't seem to narrate which tab is selected in a document
.. in a menu often just narrates "probe" instead of narrating which option in a menu was selected

**Comment by MichaelFA7** [comment]
yes, you need a document reader aloud and a dictation option, to write documents with your voice

### examples

**Out loud reading**
![Sin título](https://user-images.githubusercontent.com/126432941/224173202-9111b7f1-b0b9-41c7-abcf-ae936cb23834.png)

**Dictated documents**
![Sin título2](https://user-images.githubusercontent.com/126432941/224173864-9295bcf4-0e77-4781-b363-dcd8aa8ef1ad.png)

**Comment by Wallby** [comment]
Writing documents with your voice is a builtin feature in windows 10 and thus already possible.

**Comment by MichaelFA7** [comment]
> Escribir documentos con su voz es una función integrada en Windows 10 y, por lo tanto, ya es posible.

Yes, but what I mean is that it is natively integrated into the program, without the help of third-party programs, since not all operating systems have these options.

**Comment by MichaelFA7** [comment]
> Writing documents with your voice is a builtin feature in windows 10 and thus already possible.

Yes, but what I mean is that it is natively integrated into the program, without the help of third-party programs, since not all operating systems have these options.

**Comment by XDmitryK** [comment]
@Wallby Hi
Yeap this is known issue 52803 in our local bugtracker.

**Comment by DenisDeeSign** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too.
